pipeline/transformer/transformer_tei/TeiLocalTransformerTest.scala.txt

```diff
@@ -2,24 +2,23 @@ package weco.pipeline.transformer.tei
 
 import org.scalatest.funspec.AnyFunSpec
 import org.scalatest.matchers.should.Matchers
-import weco.storage.fixtures.S3Fixtures
-import weco.storage.store.s3.S3TypedStore
```

Comment:
Flagging for @paul-butcher's benefit: this is a test you can run against a local checkout of the TEI repo to find transformer errors. It's only updated when somebody actually uses it rather than being tested continuously, but it's quite handy when there's stuff on the DLQ and you do want to use it.